feat: in-memory state (#658)
canonical/pebble: docs/how-to/manage-a-remote-system.md

- {ref}`reference_pebble_push_command`
- {ref}`reference_pebble_rm_command`

## Run Pebble with a read-only file system

> Sometimes Pebble needs to be run in a remote system whose filesystem is read-only. For example, when Pebble is used in a Docker context with `docker run --read-only`, or in a Kubernetes context with `readOnlyRootFilesystem` set to true, mounting the container's root filesystem as read-only. In such cases, Pebble can not persist its state to the file `$PEBBLE/.pebble.state`.

Reviewer: Readability nit:
```suggestion
Sometimes Pebble needs to be run in a remote system whose filesystem is read-only. For example, when Pebble is used in a Docker context with `docker run --read-only`, or in a Kubernetes context with `readOnlyRootFilesystem` set to true, the container's root filesystem is mounted as read-only. In such cases, Pebble cannot persist its state to the state file.
```
